server/routes/user.js

```diff
@@ -6,6 +6,8 @@ const UserController = require('../controllers/UserController.js');
 router.get('/', UserController.index)
 router.patch('/', UserController.update)
 router.patch('/inventory', UserController.createInventory)
+router.patch('/coin', UserController.updateCoins)
+router.get('/coin-balance', UserController.getCoinBalance)
```

Comment:
```suggestion
router.get('/coin', UserController.getCoinBalance)
```

this might be easier to remember haha